mx.sulong/mx_sulong.py

```diff
@@ -755,7 +755,7 @@ def getLLVMVersion(llvmProgram):
     versionString = getVersion(llvmProgram)
     printLLVMVersion = re.search(r'(clang |LLVM )?(version )?(3\.\d)', versionString, re.IGNORECASE)
     if printLLVMVersion is None:
-        exit("could not find the LLVM version string in " + str(versionString))
+        exit("Could not find the LLVM version 3.x using command '" + llvmProgram + " --version'.\nYou may also use the LLVM binaries from " + _toolDir +"tools/llvm/bin" )
```

Comment:
I think we should not print that the user can use the downloaded LLVM binaries at this place. This function should only try to extract the version from the LLVM program that it receives. If the passed `llvmProgram` is `None` or specifies a non-existing path, then we have to fix this at the caller of the function.